packages/indexer-common/src/errors.ts

```diff
@@ -151,6 +153,8 @@ export const INDEXER_ERROR_MESSAGES: Record<IndexerErrorCode, string> = {
   IE066: 'Failed to allocate: allocation ID already exists on chain',
   IE067: 'Failed to query POI for current epoch start block',
   IE068: 'User-provided POI did not match reference POI from graph-node',
+  IE069: 'Failed to query Epoch Block Oracle',
```

Comment:
This error is specific to the subgraph right? 

`Failed to query Epoch Block Oracle Subgraph`?